jsonrpc/eth_endpoint.go

```diff
@@ -462,7 +462,7 @@ func (e *Eth) GetCode(address types.Address, number BlockNumber) (interface{}, e
 
 	acc, err := e.d.store.GetAccount(header.StateRoot, address)
 	if err != nil {
-		return nil, err
+		return "0x", nil
```

Comment:
We should consider wrapping this in the `arg` types, as we do with other endpoints:
`argUintPtr`, `argBigPtr`...